docs/plugins.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[kibana-plugins]]
 == Kibana Plugins added[4.2]
```

Comment:
How long do we keep this "added in previous version" stuff around?  Is a major version the right time remove it?